docs/src/add-services/varnish.md

```diff
@@ -56,20 +56,19 @@ The VCL file you provide has three specific requirements over and above the VCL
 
 The absolute bare minimum VCL file is:
 
-```bash
+```bash {location=".platform/config.vcl"}
 sub vcl_recv {
-    set req.backend_hint = application.backend();
+    set req.backend_hint = <RELATIONSHIP_NAME>.backend();
 }
 ```
 
-Where `application` is the name of the relationship defined in `services.yaml`.
-(If the relationship was named differently, use that name instead.)
+Where `<RELATIONSHIP_NAME>` is the name of the relationship you defined in [step 1](#1-configure-the-service).
 
 If you have multiple applications fronted by the same Varnish instance,
```

Comment:
:pick: That could be a subpoint to make the flow easier.

Reply:
Probably, but this isn't a rework of the doc for better structure. Should be done in #2355